packages/amplify-category-hosting/commands/hosting/remove.js

```diff
@@ -1,15 +1,84 @@
+const fs = require('fs-extra');
+const path = require('path');
+const inquirer = require('inquirer');
+
 const subcommand = 'remove';
 const category = 'hosting';
 
```

Comment:
It's not an immediate concern, but theoretically each category plugin should check what resources is managed by itself and not delete resources not managed by itself. because theoretically we can have two category plugins share the same name, and they are only distinguished by the "services" that they provide.

Reply:
This check should be part of the core theoretically and not individual platforms.